extensions/json-schema/services/src/main/resources/META-INF/cxs/schemas/items/page/page.properties.pageInfo.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$id": "https://unomi.apache.org/schemas/json/items/page/properties/pageInfo/1-0-0",
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "self": {
+    "vendor": "org.apache.unomi",
+    "name": "pageInfo",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "title": "PageInfo",
+  "type": "object",
+  "properties": {
+    "templateName": {
+      "type": ["null", "string"]
+    },
+    "language": {
+      "type": ["null", "string"],
+      "maxLength": 5
+    },
+    "destinationURL": {
+      "type": ["null", "string"],
+      "format": "uri"
+    },
+    "destinationSearch": {
+      "type": ["null", "string"]
+    },
+    "categories": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "pageID": {
+      "type": ["null", "string"]
+    },
+    "nodeType": {
+      "type": ["null", "string"]
+    },
+    "pagePath": {
+      "type": ["null", "string"]
+    },
+    "pageName": {
+      "type": ["null", "string"]
+    },
+    "referringURL": {
+      "type": ["null", "string"],
+      "format": "uri"
+    },
+    "tags": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "isContentTemplate": {
```

Comment:
I was thinking we could use extensions for non-Unomi properties such as these, what do you think ?

Reply:
This properties if necessary have to be identified and if require we will manage this in a separate PR once actions are defined on the subject.